projects/notebooks/visualizations/rhods-notebooks-performance/plotting/lts_documentation.py

```diff
@@ -82,7 +83,12 @@ def generateOneLtsDocumentationReport(entry):
     benchmark_measures += [html.Li([html.B("measures:"), html.Code(lts.results.benchmark_measures.measures)])]
 
     results += [html.Ul(benchmark_measures)]
-
     header += [html.Ul(results)]
 
+    regression = []
+    regression += [html.H2("regression analysis")]
+    regression += [html.Code(yaml.dump(entry.results.lts.regression_results, default_flow_style=False), style={"white-space": "pre-wrap"})]
```

Comment:
```suggestion
    regression += [html.H1("regression analysis")]
    regression += [html.Code(yaml.dump(lts.regression_results, default_flow_style=False), style={"white-space": "pre-wrap"})]
```
this adjusts the title size, to make it top level, and takes the `regression_results` field from `lts`, which is the focus of this file